src/browser/modules/Sidebar/ProjectFiles.tsx

```diff
@@ -209,8 +98,16 @@ const ProjectFiles = ({ bus, projectId }: ProjectFiles) => {
 
 const mapStateToProps = (state: any) => {
   return {
-    projectId: state.app.neo4jDesktopProjectId
+    projectId: getProjectId(state)
+  }
+}
+
+const mapDispatchToProps = (dispatch: any) => {
+  return {
+    resetDraft: () => {
+      dispatch(setDraftScript(null, 'project files'))
```

Comment:
could `project files` or `favorites` be assigned to an enum?

Reply:
I'll look into making it typesafe!

Reply:
Fixed in sidebarDuck!

Reply:
ok